mrp_production_category_notes/models/product.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Angel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.model
+    def _get_category_notes(self):
```

Comment:
Why is this function in the product? Wouldn't it be better to have it in the sales categories? On the other hand, is it worth saving this value and having it recalculated with depends?
Remember that there can't be a return that breaks a function; they must be allowed to finish organically.